TestVectors/p256KeyPair.json

```diff
@@ -0,0 +1,4 @@
+{
+  "publicKeyMultibase": "zDnaepBuvsQ8cpsWrVKw8fbpGpvPeNSjVPTWoq6cRqaYzBKVP",
+  "privateKeyMultibase": "z42twTcNeSYcnqg1FLuSFs2bsGH3ZqbRHFmvS9XMsYhjxvHN"
```

Comment:
I think the specs refer to this as `secretKeyMultibase` not `privateKeyMultibase`.

Reply:
Thanks for the critical read and the catching the errors. On the hash length you are right for the P384 code I switched all my hashes over to SHA-384. In the ECDSA library I use, https://github.com/paulmillr/noble-curves, you need to supply the appropriate hash algorithm corresponding to the curve used so I switched all of them over. 

We discussed this a  bit in https://github.com/w3c/vc-di-ecdsa/pull/12. So we should either update the algorithm description (my preference and makes for happier crypto reviewers) or change the P384 test vectors. Thoughts? I can do the PR.

Cheers Greg

Reply:
@Wind4Greg,

> you need to supply the appropriate hash algorithm corresponding to the curve used so I switched all of them over.

I definitely think the hash used to sign should be SHA-384 (when using P-384). It would definitely harm interop if we did something else. I don't think anyone has given a reason to do anything else there.

However, I don't think we should change the SHA-256 hashes used to prepare the message that will be signed (which is a 64-byte concatenation of two SHA-256 digests of canonicalized values). That hashing and entire process happens at a different layer from the digital signature; it isn't related to the signing algorithm (or the signing algorithm's related hash mechanism). I think it will complicate software to change that and I don't yet think there's much value in doing it.

Reply:
The reason we use SHA-256 with Ed25519 is that we want to achieve 128 bits of security for the entire system which includes the hashing of the options and the  document prior to their concatenation and the signing.

For system wide 192 bits of security it would be expected that all hashing would be at the SHA-384 level then a 192 bit signature scheme such as ECDSA with P-384. Each piece is expected to be at the level of 192 bits. See Boneh and Shoups online book or their Coursera course. They gauge the security level by the weakest link in the chain.

Or we can drop ECDSA with P-384 if folks don't want it.

Reply:
@Wind4Greg,

I agree that to avoid any controversy we should use SHA-384 when computing the message to be signed if we're going to support P-384. So, provided that we continue on supporting P-384 -- we should go with the test vectors you generated here.